### ISSUE TITLE
Timer is not starting at 0.00 

### ISSUE THREAD

**Issue by Sacricx** [shavitush/bhoptimer#1010]
Hello, 
whenever people don't leave the startzone in the very last moment, their timer is already at 0.7 seconds or similar depending on how far away from end of the beam they jump off the startzone. (Prespeed Setting No. 3)

Many people (don't ask me why) prefer to prejump (limited to prestrafe limit) before leaving the startzone. Whenever they do, their timer is already at 2.0 seconds or similar when leaving the startzone. (Prespeed setting No. 4)

My impression is: Once you lift from the ground with a jump, the timer is starting already (even within the startzone). Once you land back on the ground it resets (or adds up) the time and starts again as soon as you leave the ground. Even by doing only one jump and jumping off the "middle" of the startzone, the timer is counting your air time (flying out of the startzone) already as runtime.

How it should be: 
No matter from where people do jump off the startzone, the timer should only start when leaving the "Box" and not on the first, second or third jump (depending on the prespeed setting).

Here is an example video of me jumping off the startzone 3 times in the very last moment (timer starts at 0.00) 
Then 3 times jumping of the "middle" off the startzone into the run 0.7. And then 3 times with prejump and the timer is (not visible within the startzone) starting at around 2.0.

[https://youtu.be/Ygl3hL3JOdk](url)

TL;DR
The timer is counting airtime within the startzone already as runtime. It only starts from 0.00 when leaving the startzone at the very end of the "box".

best regards and thanks in advance 

Sacricx

**Comment by hermansimensen** [comment]
There is already a setting to fix this. It's called NoZAxisSpeed and found in one of the configs.

**Comment by Sacricx** [comment]
Fast and precise !
Thank you, this solved my issue ;)
Im using this timer for a long period of time now. I don't know why this simple setting didn't got my attention, when going through the settings.

Again, thank you very much for solving this issue for me.

Best Regards

Sacricx
